code/swb_base/ui/SniperScope.cs

```diff
@@ -36,7 +36,8 @@ public override void Tick()
     {
         base.Tick();
 
-        if (Game.LocalPawn is not PlayerBase player) return;
+        if (Game.LocalPawn is not Entity ent) return;
+        if (Game.LocalPawn is not ISWBPlayer player) return;
```

Comment:
Why do you check for both here, wouldn't the ISWBPlayer check be enough?

Reply:
This can be fixed by making `ISWBPlayer` extend from `IEntity` which I will do.